custom-words.txt

```diff
@@ -1623,6 +1623,7 @@ mlflowtrackinguri
 mltable
 mmtc
 mnist
+mobo
```

Comment:
would be great if we don't need to add this in this PR :)

Reply:
Its to resolve the validation error to add to custom words. MoboBrokers is agreed upon and approved name for a resource type where it already has the api spec.